liquibase-standard/changelog-with-rollback.yaml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
```

Comment:
Same case as what I commented below for `test-changelog-test-clob...`.

Reply:
I actually didn't create this file myself, I'm not sure if it was created as part of the tests and I stopped the execution before it was deleted, or if I accidentally copied it from somewhere. I'll just delete it.